### ISSUE TITLE
Istio and Dev Spaces

### ISSUE THREAD

**Issue by brandondjmurphy** [Azure/dev-spaces#42]
I have a vanilla cluster with Istio installed, and when I install Dev Spaces, then all routing within the cluster stops working. It appears that Dev Spaces is conflicting with Istio. When I uninstall Dev Spaces from the cluster, then Istio routing works perfectly.

Is this a known issue?

**Comment by stepro** [comment]
Yes, it's a known issue that Istio and Dev Spaces do not currently play well together. We'd like to make this work, however. It would be helpful to know if you are using the routing capabilities of Dev Spaces in addition to Istio and what behavior you would expect in this case.

**Comment by irperez** [comment]
In our case, I would expect that we could leverage routing and security aspects of Istio in Dev spaces.  This would make for less changes between development and production.
